scala/src/main/scala/raytracer/Image.scala

```diff
@@ -20,7 +20,7 @@ object Image {
   def apply(width: Int, height: Int, pixel: (Int, Int) => Pixel): Image = {
     val arr = ParSeq.tabulate(width * height)(n =>
         (height - n / height, n % width))
-    val pixelData = arr.map(pixel.tupled)
+    val pixelData = arr.par.map(pixel.tupled)
```

Comment:
I am pretty sure that `.par` does nothing on a `ParSeq` and is only there for compatibility with the old collections.

Reply:
Indeed, I revert it :)